sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.4.0-beta.1 (Unreleased)
+## 1.4.0-beta.1 (2022-02-03)
 
 ### Features Added
```

Comment:
@antkmsft is this really a new feature? This seemed more like a bug fix, or an SDK infra change that might fit into "Other changes" category. Up to you, just curious.

Reply:
I can see how this can be viewed as a bug fix, but I think of it more as a as a feature, because previously, if you set `AZURE_LOG_LEVEL`, on UWP, there would be no effect, and now there is. And our code previously would explicitly comment out the pieces that do environment, just because we thought it is not possible to have env on UWP at all - not like we had it, and were surprised to find it does not work, the Azure SDK simply wouldn't link if you use `std::getenv`.

Reply:
@ahsonkhan, but in terms of "would you increment minor version for this" - I think rather not.

Reply:
> @ahsonkhan, but in terms of "would you increment minor version for this" - I think rather not.

This is where I was thinking, we maybe move this to bug fix. It isn't really something customers would need to change their code on or adopt to get the benefit of, unlike new APIs/clients, or features (for example, going from no logging to adding logging could be considered a feature because it enables a larger scenario).